.github/labeler.yaml

```diff
@@ -0,0 +1,40 @@
+CI / testing:
+  - all: ['.github/**', '**/test_*']
+
+car: 
+  - all: ['selfdrive/car/**']
+
+body:
+  - all: ['selfdrive/car/body/*']
+chrysler:
+  - all: ['selfdrive/car/chrysler/*']
+ford: 
+  - all: ['selfdrive/car/ford/*']
+gm: 
+  - all: ['selfdrive/car/gm/*']
+honda: 
+  - all: ['selfdrive/car/honda/*']
+hyundai: 
+  - all: ['selfdrive/car/hyundai/*']
+mazda: 
+  - all: ['selfdrive/car/mazda/*']
+nissan: 
+  - all: ['selfdrive/car/nissan/*']
+subaru: 
+  - all: ['selfdrive/car/subaru/*']
+tesla: 
+  - all: ['selfdrive/car/tesla/*']
+toyota: 
+  - all: ['selfdrive/car/toyota/*']
+volkswagen: 
+  - all: ['selfdrive/car/volkswagen/*']
+
+simulation:
+  - all: ['tools/sim/**']
+ui:
+  - all: ['selfdrive/ui/**']
+tools: 
+  - all: ['tools/**']
+
+#multilanguage: selfdrive/ui/translations/**
```

Comment:
should be able to do this one too

Reply:
probably all except main_en?

Reply:
can't we just do all in translations/**?

Reply:
ah, yes nvm